src/component/Allproperties/Allproperties.jsx

```diff
@@ -0,0 +1,349 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import Footer from "../Footer/Footer";
+import NavBar from "../NavBar/NavBar";
+import axios from "axios";
+import "./Allproperties.css";
+import { setCurrentUser } from "../../slice/propertyslice";
+
+export const Propertydata = [
```

Comment:
Since your data is coming from the API, there shouldn't be a need for this propertyData array anymore.

Reply:
worked on boss